com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md

```diff
@@ -79,6 +79,10 @@ These settings control the draw distance and resolution of the decals atlas that
 | **Property**                    | **Description**                                              |
 | ------------------------------- | ------------------------------------------------------------ |
 | **Enable**                      | Enable the checkbox to make HDRP support dynamic resolution in your Unity Project. |
+| **- Enable DLSS**               | Enable the checkbox to make HDRP support NVIDIA Deep Learning Super Sampling (DLSS).<br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project. |
+| **-- Performance Quality**      | Use the drop-down to select which performance mode DLSS operates on. The options are:<br/>&#8226; **Balanced**: - Balances performance with quality.<br/>&#8226; **MaxPerf**: - Fast performance, lower quality.<br/>&#8226; **MaxQuality**: - High quality, lower performance.<br/>&#8226; **UltraPerformance**: - Fastest performance, lowest quality.<br/>&#8226; **UltraQuality**: - Highest quality, lowest performance. |
+| **-- Use Optimal Settings**     | Enable the checkbox to make DLSS control the Sharpness and Screen Percentage automatically. |
+| **-- Sharpness**                | Use the slider to set the pixel sharpness of the DLSS upscaler. |
```

Comment:
I suggested another tooltip on editor side to prevent us using the same word (shaprness) we're trying to define and give a bit more info on what's at stake here. 

"Controls how the DLSS upsampler will render edges on the image. More sharpness usually means more contrast and clearer image but can increase flickering and fireflies. This setting is ignored if ‘Use Optimal Settings’ is checked."